src/index.js

```diff
@@ -2,16 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
+import Parent from './ParentComponent';
 import reportWebVitals from './reportWebVitals';
 
 ReactDOM.render(
+  <React.Fragment>
```

Comment:
<React.Fragment> should be removed from here. It also have a short syntax <> </>

Reply:
Changes are done.